docs/plugins/convert.rst

```diff
@@ -111,6 +111,8 @@ file. The available options are:
   This option overrides ``link``. Only works when converting to a directory
   on the same filesystem as the library.
   Default: ``false``.
+- **delete_originals**: Transcoded files will be copied or moved to their destination, depending on the import configuration. By default, the original files are not modified by the plugin. This option deletes the original files after the transcoding step has completed.
```

Comment:
#2947 seems to suggest that currently if you have `import: move` enabled in your config, then the convert plugin will not "move" the converted files (and delete the originals). This PR seems to just add a flag on whether or not to delete the original files. 

Basically, does this PR change the behavior of this plugin to match this:

>  Transcoded files will be copied or moved to their destination, depending on the import configuration.

Reply:
Good question.

The original PR did what you say - if you have `import: move` enabled, the convert plugin deletes the original files. However, the comments in that PR pointed out that this may not be desirable - an example given was that when transcoding from a lossless format to lossy format, the user may wish to keep the original files for archiving. Removing the source files might be surprising to the user, and is irreversible.

I found that argument compelling. Imho, it makes sense that the `import: move` command should refer to what happens to the _transcoded_ files - they would either be copied or moved into the library in accordance with this setting.

In this PR, if the user wishes to move the transcoded files into the library and to remove the source files after conversion (as I do), it would be necessary to specify both `import: move: yes` and `convert: delete_originals: yes`. This extra parameter seems to me to be a suitable compromise between arriving at the desired functionality while not causing surprising data loss for users that don't want this behaviour.

Reply:
Thanks for the clarification! I agree, your approach makes the most sense.